src/LiveQuery/ParseLiveQueryServer.ts

```
@@ -1056,6 +1081,192 @@
+
+    const resp: any = {
+      results: this.replacePointers(response.results, path, replace),
+    };
+    if (response.count) {
+      resp.count = response.count;
+    }
+    return resp;
+  }
+
+  findPointers(object: any, path: Array<string>): any[] {
+    if (object instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)